index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Website</title>
+</head>
+<body>
+    <header class="container">
+        <a class="logo">My Logo</a>
+
+        <nav class="menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </nav>
+    </header>
+
+    <section class="heading">
+        <h1 class="title">I’m Shaheda
+            Glad to see you!</h1>
+
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+    <p class="line">LET’S CONNECT</p>
+    
+    <div class="social-media-icons">
+        <a href=""><img class="twitter" src="src/Twitter.png" alt="twitter-icon"/></a>
+
+        <a href=""><img class="linkedin-icon" src="src/Linkedin-icon.png" alt="linkedin-icon"/></a>
+
+        <a href=""><img class="msign" src="src/M.png" alt="msign"/></a>
+
+        <a href=""><img class="github" src="src/Github.png" alt="github-icon"/></a>
+
+        <a href=""><img class="finger" src="src/Finger.png" alt="Finger-icon"/></a>
```

Comment:
- [ ] Your social media heading and social media icons are aligned differently from the design, Please update your design and make sure it matches the design on the template.

Your design

![Screenshot (79)](https://user-images.githubusercontent.com/83803414/153583613-c24090ca-c6de-4cc9-a630-1b41ce5b74d4.png)


Figma design

![Screenshot (80)](https://user-images.githubusercontent.com/83803414/153583594-61f9c708-3259-45b6-b230-aaae4e17affd.png)